### ISSUE TITLE
Cannot set nullable to null in debugger

### ISSUE THREAD

**Issue by Thaina** [Unity-Technologies/vscode-unity-debug#137]
From debugging session

> round.GetType()
> {System.Nullable`1[System.Int32]}
> round = null
> Evaluate request failed (invalid expression).

Suspect that it was from here

https://github.com/microsoft/vscode-mono-debug/blob/21a0b1433e8ca281fd1f9fc15ba395ef33ed2686/src/MonoDebugSession.cs#L730-L763

Is this a problem of debugger lib repo?

**Comment by miniwolf** [comment]
Yes that is a problem in the debugger libs. But I do not have the high hopes for them to solve it. We are using a fork of the debugger-libs so we can potentially go in a fix this problem ourselves. If you want to investigate more you can clone the repo and submodules and see if you can come up with something.
Regardless I will put this is as bug/missing feature, so I can check it when I get time to upgrade the libraries.

**Comment by Thaina** [comment]
> We are using a fork of the debugger-libs

Did the problem on debugging async/await also caused by main repo or it is a problem from forked repo?

**Comment by miniwolf** [comment]
async/await is no correctly supported on the main repo either.
